uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/apis/entitlement-api.js

```diff
@@ -0,0 +1,88 @@
+define([
+  'leave-absences/shared/modules/apis',
+  'common/services/api'
+], function (apis) {
+  'use strict';
+
+  apis.factory('EntitlementAPI', ['$log', 'api', function ($log, api) {
+    $log.debug('EntitlementAPI');
+
+    return api.extend({
+
+      /**
+       * This method returns all the entitlements (no pagination required), but optionally
+       * it can chain `Entitlement.remainder` (PCHR-1132) to return also the balance (current and future)
+       * among the rest of the data.
+       *
+       * If chaining, it should normalize the result by removing the `Entitlement.balance` property
+       * from the result, so that the Model doesn't have to use a property that is specific to the
+       * current API implementation
+       *
+       * @param  {Object} params      matches the api endpoint params (period_id, contact_id, etc)
+       * @param  {boolean} withRemainder
+       * @return {Promise}
+       */
+      all: function (params, withRemainder) {
+        $log.debug('api.leave-absences.entitlement.all');
+
+        var params = {};
+        /*params = {
+          sequential: 1
+        }*/
+
+        if (withRemainder) {
+          params['api.LeavePeriodEntitlement.getremainder'] = {
+            "entitlement_id": "$value.id",
+            "include_future": true
+          }
+        }
+
+        return this.sendGET('LeavePeriodEntitlement', 'get', params)
+          .then(function (data) { return data.values; })
+          .then(function (entitlements) {
+            if (withRemainder) {
+              /* normalize the results
+              {
+                id: '1',
+                contract_id: '2',
+                type_id: '4',
+                //...
+                balance: {
+                  current: 20,
+                  future: 10
+                }
+              }
+              */
+              //the data will have key 'api.LeavePeriodEntitlement.getremainder'
```

Comment:
changed it to 
` 
             //entitlements data will have key 'api.LeavePeriodEntitlement.getremainder'
              //which is normalized with a friendlier 'remainder' key
`